kolibri/plugins/learn/assets/src/composables/useDownloadRequests.js

```diff
@@ -35,7 +35,7 @@ const availableSpace = ref(0);
 export default function useDownloadRequests(store) {
   store = store || getCurrentInstance().proxy.$store;
 
-  const { instanceId } = useDevices(store);
+  const { instanceId } = currentDeviceData(store);
 
   function fetchUserDownloadRequests(params) {
     return ContentRequestResource.list(params)
```

Comment:
I can't comment on the actual line, but please update the setting of the downloadRequestMap to `obj.contentnode_id` not `obj.id`.